db/models/assessment_record/assessment_records.py

```diff
@@ -0,0 +1,82 @@
+from sqlalchemy import Index
+from db import db
+from db.models.assessment_record.enums import Status
+from sqlalchemy import cast
+from sqlalchemy import Computed
+from sqlalchemy import func
+from sqlalchemy.dialects.postgresql import ENUM
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.dialects.postgresql import TEXT
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class AssessmentRecords(db.Model):
+    __tablename__ = "assessment_records"
+
+    application_id = db.Column(
+        "application_id", UUID(), primary_key=True
+    )
+
+    short_id = db.Column("short_id", db.Text(), nullable=False)
```

Comment:
Is this meant to be the friendly application reference, if so this could be a String type rather than text,